inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongClusterEntityMapper.java

```diff
@@ -57,4 +57,6 @@ List<InlongClusterEntity> selectByKey(@Param("clusterTag") String clusterTag, @P
 
     int deleteByPrimaryKey(Integer id);
 
+    List<InlongClusterEntity> selectByClusterTag(String clusterTag);
```

Comment:
Please move the `select` method after the `selectAllClusters()` method.

Reply:
fixed, thx